docs/book/src/migration/v1vsv2.md

```diff
@@ -0,0 +1,66 @@
+# Kubebuilder v1 vs v2
+
+This document cover all breaking changes when migrating from v1 to v2.
+
+The details of all changes (breaking or otherwise) can be found in
+[controller-runtime](https://github.com/kubernetes-sigs/controller-runtime/releases),
+[controller-tools](https://github.com/kubernetes-sigs/controller-tools/releases)
+and [kubebuilder](https://github.com/kubernetes-sigs/kubebuilder/releases)
+release notes.
+
+## Common changes
+
+- All three projects now use `go modules` instead of `dep`. Dep support in
+`controller-runtime` will be maintained until the release of Go 1.13 (when
+modules are enabled by default).
+
+## controller-runtime
+
+- `Client.List` now uses functional options (`List(ctx, list, ...option)`) instead
+of `List(ctx, ListOptions, list)`.
+- `Client.DeleteAllOf` was added to the `Client` interface.
+
+- Metrics are on by default now.
+
+- A number of packages under `pkg/runtime` have been moved, with their old
+locations deprecated. The old locations will be removed before
+controller-runtime v1.0.0. See the [godocs][pkg-runtime-godoc] for more
+information.
+
+#### Webhook-related
+
+- Automatic certificate generation for webhooks has been removed, and webhooks
+will no longer self-register. Use controller-tools to generate a webhook
+configuration. If you need certificate generation, we recommend using
+[cert-manager](https://github.com/jetstack/cert-manager). Kubebuilder v2 will
+scaffold out cert manager configs for you to use -- see the
+[Webhook Tutorial](/cronjob-tutorial/webhook-implementation.md) for more details.
+
+- The `builder` package now has separate builders for controllers and webhooks,
+which facilitates choosing which to run.
+
+## controller-tools
```

Comment:
Capitalize it ? `Controller tools`

Reply:
Discussed offline and decided to keep as is.